src/index.html

```diff
@@ -1,17 +1,210 @@
+            <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            >
+          </label>
+        </div>
+
+        <div class="input password">
+          <label for="password">Password:
+            <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="6"
+            maxlength="15"
+            >
+          </label>
+        </div>
+
+      </fieldset>
+
+      <fieldset class="form-field hobby">
+        <legend>An interesting fact about you!</legend>
+        <div class="input cats">
+          <label for="cats">Do you love cats?
+            <input
+            type="radio"
+            name="cats"
+            id="cats yes"
+            >Yes
+
+            <input
+            type="radio"
+            name="cats"
+            id="cats no"
+            >No
+          </label>
+        </div>
+
+        <div class="input color">
+          <label for="color">What is your favorite color?
+            <input
+            type="color"
+            name="color"
+            id="color"
+            >
+          </label>
+        </div>
+
+        <div class="input sleep">
+          <label for="sleep">What time do you go to bed?
+            <input
+            type="time"
+            name="sleep"
+            id="sleep"
+            >
+          </label>
+        </div>
+
+        <div class="input cars">
+          <label for="cars">What are your favorite brand of cars?
+            <select
+            name="cars"
+            id="cars"
+            multiple>
+              <option value="bmw">BMW</option>
+              <option value="audi">Audi</option>
+              <option value="lada">Lada</option>
+            </select>
+          </label>
+        </div>
+
+        <div class="input rating">
+          <label for="rating">How do you rate our work?
+            <input
+            type="range"
+            name="rating"
+            id="rating"
+            value="0"
+            >
+          </label>
+        </div>
+
+      </fieldset>
+
+      <fieldset class="form-field info">
+        <legend>Additional info:</legend>
+        <div class="input comments">
+          <label for="comments">Comments:
+            <textarea
+            name="comments"
+            autocomplete="off"
+            id="comments"
+            cols="20"
+            rows="2"
+           ></textarea>
+          </label>
+        </div>
+
+        <div class="input recomendation">
+          <label for="recomendation">Would you recomend us?
+            <select
+            name="recomendation"
+            id="recomendation">
+              <option value="yes">Yes</option>
+              <option value="no">No</option>
+            </select>
+          </label>
+        </div>
+      </fieldset>
+
+      <div class="input submit">
+        <label for="submit">
+          <input
```

Comment:
This element should be a button of type submit and without the unnecessary wrapping of div and label